api/wd/index.md

```diff
@@ -210,35 +210,45 @@ browser
   .waitForUrl('/products', { start: 30, count: 15 });
 ```
 
-### Elements
+## Elements
 
-#### `browser.clickOn(cssSelector)`
+### `browser.clickOn(cssSelector)`
 
-Calls `clickOn()` on the Element found by the given `cssSelector`.
+Calls `clickOn()` on the element found by the given `cssSelector`.
```

Comment:
```suggestion
Calls `click()` on the Element found by the given `cssSelector`.
```

I like the capitalization because it shows that it's operating on an instance of our Element class, but the correct method name for that is `.click()`